server/bin/gold/test-2.txt

```diff
@@ -89,7 +89,10 @@ len(actions) = 1
 ]
 --- Finished pbench-dispatch (status=0)
 +++ Running pbench-unpack-tarballs small
-pbench-unpack-tarballs: Bad INCOMING=/var/tmp/pbench-test-server/test-2/pbench/public_html/incoming
+pbench-unpack-tarballs-small: Bad INCOMING=/var/tmp/pbench-test-server/test-2/pbench/public_html/incoming
+--- Finished pbench-unpack-tarballs (status=1)
```

Comment:
That's interesting: why did this change from `pbench-unpack-tarballs` to `pbench-unpack-tarballs-small`?? Or rather, it looks like it ought to have been in the first place, but why did it change? Ah; you moved the `PROG` renaming up before the config errors were detected. Cool.